backend/app/usecase/changelog/changelog.go

```diff
@@ -0,0 +1,46 @@
+package changelog
+
+import (
+	"github.com/short-d/app/fw"
+	"github.com/short-d/short/app/usecase/keygen"
+
+	"github.com/short-d/short/app/entity"
+	"github.com/short-d/short/app/usecase/repository"
+)
+
+type ChangeLog interface {
+	CreateChange(title string, summaryMarkdown string) (entity.Change, error)
+	GetChangelog() ([]entity.Change, error)
+}
+
+type Persist struct {
+	keyGen        keygen.KeyGenerator
+	timer 		  fw.Timer
+	changelogRepo repository.Changelog
```

Comment:
The reason we had all lowercase for package name is because that's Golang's convention. We use camelcase everywhere else to make the words sensitive to eyes